### PR TITLE
osm: HOT, cylclemap, transportation

### PR DESCRIPTION
**Pull request by blackgis**
Tiles from http://www.openstreetmap.org/

### CODE REVIEW

File: data_sources/osm_cyclemap/metadata.ini [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+[license]
+name = CC-BY-SA 2.0
+link = http://creativecommons.org/licenses/by-sa/2.0/
+copyright_text = © OpenStreetMap contributors, CC-BY-SA
+copyright_link = https://www.openstreetmap.org/copyright
+terms_of_use = http://wiki.openstreetmap.org/wiki/Tile_usage_policy
+
+[tms]
+url = http://a.tile.thunderforest.com/cycle/{z}/{x}/{y}.png
```
Comment (yellow-sky) [inline data_sources/osm_cyclemap/metadata.ini:18]: Already exists - https://github.com/nextgis/quickmapservices/blob/master/src/data_sources/osm_tf_opencyclemap/metadata.ini

File: data_sources/osm_humanitarian/metadata.ini [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+[license]
+name = CC-BY-SA 2.0
+link = http://creativecommons.org/licenses/by-sa/2.0/
+copyright_text = © OpenStreetMap contributors, CC-BY-SA
+copyright_link = https://www.openstreetmap.org/copyright
+terms_of_use = http://wiki.openstreetmap.org/wiki/Tile_usage_policy
+
+[tms]
+url = http://tile-b.openstreetmap.fr/hot/{z}/{x}/{y}.png, http://tile-a.openstreetmap.fr/hot/{z}/{x}/{y}.png, http://tile-c.openstreetmap.fr/hot/{z}/{x}/{y}.png
```
Comment (yellow-sky) [inline data_sources/osm_humanitarian/metadata.ini:18]: Only one url, please

File: data_sources/osm_standard/metadata.ini [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+[license]
+name = CC-BY-SA 2.0
+link = http://creativecommons.org/licenses/by-sa/2.0/
+copyright_text = © OpenStreetMap contributors, CC-BY-SA
+copyright_link = https://www.openstreetmap.org/copyright
+terms_of_use = http://wiki.openstreetmap.org/wiki/Tile_usage_policy
+
+[tms]
+url = http://a.tile.openstreetmap.org/{z}/{x}/{y}.png
```
Comment (yellow-sky) [inline data_sources/osm_standard/metadata.ini:18]: Already exists - https://github.com/nextgis/quickmapservices/blob/master/src/data_sources/osm_mapnik/metadata.ini

File: data_sources/osm_transportation/metadata.ini [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+[license]
+name = CC-BY-SA 2.0
+link = http://creativecommons.org/licenses/by-sa/2.0/
+copyright_text = © OpenStreetMap contributors, CC-BY-SA
+copyright_link = https://www.openstreetmap.org/copyright
+terms_of_use = http://wiki.openstreetmap.org/wiki/Tile_usage_policy
+
+[tms]
+url = http://a.tile.thunderforest.com/transport/{z}/{x}/{y}.png
```
Comment (yellow-sky) [inline data_sources/osm_transportation/metadata.ini:18]: Already exists - https://github.com/nextgis/quickmapservices/blob/master/src/data_sources/osm_tf_transport/metadata.ini

**Comment by yellow-sky** [comment]
Please send new PR with one correct datasource - Humanitarian

File: data_sources/osm_transportation/metadata.ini (hunk 4 above)
Comment (blackgis) [inline data_sources/osm_transportation/metadata.ini:18]: Transportation is the official naming!

File: data_sources/osm_standard/metadata.ini (hunk 3 above)
Comment (blackgis) [inline data_sources/osm_standard/metadata.ini:18]: Standard is the official naming!

File: data_sources/osm_humanitarian/metadata.ini (hunk 2 above)
Comment (blackgis) [inline data_sources/osm_humanitarian/metadata.ini:18]: It works with multiurl! I did a test on this!

File: data_sources/osm_cyclemap/metadata.ini (hunk 1 above)
Comment (blackgis) [inline data_sources/osm_cyclemap/metadata.ini:18]: Official naming for Cycle Map

**Comment by blackgis** [comment]
@yellow-sky 

> Please send new PR with one correct datasource - Humanitarian

But Multiurl works! Seen from Openlayers and sucessfully tested in qms. Just merge this and its fine.

Best, Richard.

**Comment by simgislab** [comment]
Ok. I manually moved most of the changes (official names etc.), I also added humanitarian with 3 tile sources (tried it and it works), lets see what happens. Closing this for now.

**Pull request closed without merging**